ourjapanlife/findadoc-server
# 104 insurance seed data

Resolves #104

# What changed
- Use seed data for loading insurance
- Fix migration to remove `UNKNOWN` from enum (empty will be unknown)
- Fix an API test

# Testing instructions
1. In one terminal, start the db with `docker compose up`
2. Ensure the seeding completes 
```sh
findadoc-server-migrate-1       | {
findadoc-server-migrate-1       |   id: 2,
findadoc-server-migrate-1       |   nameJa: '博士号',
findadoc-server-migrate-1       |   nameEn: 'Doctor of Philosophy',
findadoc-server-migrate-1       |   abbreviation: 'PhD'
findadoc-server-migrate-1       | }
findadoc-server-migrate-1       |
findadoc-server-migrate-1       | 🌱  The seed command has been executed.
```

3. In another terminal, open prisma studio with `yarn prisma studio`
4. Look at the healthcare professionals and inspect the "acceptedInsurance []" and verify that the seeded data has some variable examples:

<img width="1416" alt="Prisma_Studio" src="https://user-images.githubusercontent.com/4602369/215262832-ef77bbb1-5c72-464f-96f4-0e1bbc1c49e5.png">

I also fixed a test as part of this PR... perhaps I should have broken it out, but you can run it with `yarn test`
